tests/integration/cli/validate/test_cli_validate.py

```diff
@@ -48,90 +52,166 @@ def test_validate_otm_yaml_file(self, caplog):
+            # Given a list of arguments with
+            args = [
+                # a valid mapping type
+                # and without mapping-file argument
+                '--mapping-type', 'MTMT']
 
-        # Then validator returns not OK
-        assert result.exit_code == 1
-        #   and validation ETM is performed
-        assert 'Validating External Threat Model mapping files' in caplog.text
-        #   and validation ETM finish unsuccessfully
-        assert 'Mapping file is not valid' in caplog.text
-        assert '\'label\' is a required property' in caplog.text
+            # When validating
+            result = runner.invoke(validate, args)
 
-    def test_validate_multiple_files(self, caplog):
-        # Given a info level of logging
-        caplog.set_level(logging.INFO)
+            # Then the exit code is 2
+            assert result.exit_code == 2
+            assert result.stdout.__contains__("Error: Missing one of this arguments: {")
+            assert result.stdout.__contains__("mapping_file")
+            assert result.stdout.__contains__("otm_file")
+
+    def test_validate_command_without_type_argument(self):
         runner = CliRunner()
 
-        # When validating multiple files
-        result = runner.invoke(validate, [
-            '--otm-file', SAMPLE_OTM_FILENAME,
-            '--iac-mapping-file', IAC_VALID_MAPPING_FILENAME,
-            '--diagram-mapping-file', CUSTOM_YAML_VISIO_MAPPING_FILENAME
-        ])
+        with runner.isolated_filesystem():
+            # Given a list of arguments with
+            args = [
+                # a valid mapping file
+                # and without mapping-type argument
+                '--mapping-file', CFT_VALID_MAPPING_FILENAME]
+
+            # When validating
+            result = runner.invoke(validate, args)
+
+            # Then the exit code is 2
+            assert result.exit_code == 2
+            assert result.stdout.__contains__("Error: Missing one of this arguments: {")
+            assert result.stdout.__contains__("mapping_type")
+            assert result.stdout.__contains__("otm_file")
+
+    @pytest.mark.parametrize('mapping_file, mapping_file_type', [
+        pytest.param(MTMT_MAPPING_FILENAME_VALID, 'CLOUDFORMATION', id="with a valid MTMT mapping file and an invalid "
+                                                                       "CLOUDFORMATION type"),
+        pytest.param(VISIO_VALID_MAPPING_FILE, 'TERRAFORM', id="with a valid VISIO mapping file and an invalid "
+                                                               "TERRAFORM type"),
+        pytest.param(CFT_VALID_MAPPING_FILENAME, 'VISIO',
+                     id="with a valid CLOUDFORMATION mapping file and an invalid VISIO type"),
+        pytest.param(TF_VALID_MAPPING_FILE, 'MTMT', id="with a valid TERRAFORM mapping file and an invalid MTMT type")])
+    def test_validate_valid_mapping_file_and_invalid_type(self, mapping_file, mapping_file_type):
```

Comment:
Should we rename this test to "test_validate_valid_mapping_file_and_wrong_type", and create a specific test for invalid types (formats that are not implemented)?

Reply:
A new test for testing the wrong mapping types has been added.